.gitignore

```diff
@@ -1,2 +1,5 @@
 out
-node_modules
\ No newline at end of file
+node_modules
+package-lock.json
```

Comment:
The lock file shouldn't be gitignored, since it provides reproducibility of builds (https://medium.com/coinmonks/everything-you-wanted-to-know-about-package-lock-json-b81911aa8ab8)

Reply:
Not sure why the lock file isn't already in the repo, guessing it's because I was using an pre 5.x version of npm.